backend/src/recommitment/recommitment.service.ts

```diff
@@ -84,12 +84,43 @@ export class RecommitmentService {
       program: key,
     });
 
+    const otherProgram = key === Program.BCWS ? Program.EMCR : Program.BCWS;
+    const otherProgramPersonnel = otherProgram === Program.BCWS ?
+      await this.bcwsService.getBcwsPersonnelById([Role.COORDINATOR], id) :
+      await this.emcrService.getEmcrPersonnelById([Role.COORDINATOR], id);
+    let otherProgramStatusReset = false;
+    if (otherProgramPersonnel) {
+      const recommitmentOtherProgram = await this.recommitmentRepository.findOneByOrFail({
+        personnelId: id,
+        recommitmentCycleId: new Date().getFullYear(),
+        program: otherProgram,
+      });
+      // Unless they were already approved for the other program, reset to PENDING
+      if (recommitmentOtherProgram.status !== RecommitmentStatus.SUPERVISOR_APPROVED) {
```

Comment:
Are we still showing the recommitment reinit button on the FE if a user becomes INACTIVE for other reasons, but had been prev. approved by their supervisor?

Reply:
I think so, yes. We leave it up to the Coordinator which pathway they want to go through for the member's reactivation